docs/render-pipeline/backends.md

```
@@ -1,24 +1,87 @@
+    pub height:  u32,
+    pub data:    Arc<Vec<u8>>,   // premultiplied BGRA32, stride = width × 4
+}
+```
+
+`Arc` ownership lets the host thread hold tiles across frames with zero copies, even while the engine is building the next cache.
+
+### TileCache scroll fields
+
+`scroll_x` / `scroll_y` in `TileCache` record the engine's scroll offset at the time the handle was created. The host uses them to compute screen positions:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)